cypress/integration/edit_lay_percentage.spec.js

```diff
@@ -0,0 +1,13 @@
+describe("Edit lay percentage", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/random-reviews/edit/lay-percentage");
+    });
+
+    it("allows me to change the lay percentage value", () => {
```

Comment:
This doesn't appear to describe what is being tested. You are testing that an error message displays if you input the incorrect value.

Reply:
done